test/extended/util/image/image.go

```diff
@@ -150,6 +152,12 @@ func OriginalImages() map[string]int {
 	return images
 }
 
+// Exceptions is a list of images we don't mirror temporarily due to various
+// problems. This list should ideally be empty.
+var Exceptions = sets.NewString(
```

Comment:
how did we find this?

Reply:
When you follow https://github.com/openshift/origin/blob/master/test/extended/util/image/README.md and try to mirror the mirroring didn't work. I tried different target registry (namely docker.io and it worked there). This is a new image, related with windows testing I've put that exception there until quay allows us mirroring windows images.